styles/status.cafe/catppuccin.user.css

```diff
@@ -0,0 +1,187 @@
+        @mantle: @catppuccin[@@lookup][@mantle];
+        @crust: @catppuccin[@@lookup][@crust];
+        @accent-color: @catppuccin[@@lookup][@@accent];
+
+        color-scheme: if(@lookup = latte, light, dark);
+
+        ::selection {
+            background-color: fade(@accent-color, 30%);
+        }
+
+        input,
+            textarea {
+                &::placeholder {
+                color: @subtext0 !important;
+            }
+        }
+
+        /* Start styling your website here: */
+        background-color: @base;
+
+        body {
+            background-color: @base;
+            color: @text;
+        }
+        
+        textarea {
+            background-color: @mantle;
+            border: 1px solid @overlay2;
+            border-radius: 2px;
+            padding: 3px;
+        }
+        
+        input {
+            background-color: @crust;
+            border: 1px solid @overlay2;
+            border-radius: 4px;
+            padding: 2px 5px 2px 5px;
+        }
+        
+        input#homepage, input#email, input#picture {
+                background-color: @mantle !important;
+            }
+
+        .flash {
+            background-color: @green;
+        }
+
+        /*accent color links*/
+        & when (@colors = 1) {
+            a:not(.flash a) {
+                color: @accent-color;
+            }
+
+            a:visited:not(.flash a) {
+                color: @overlay0;
+            }
+
+            /* visited colors*/
+            & when (@accent = rosewater), (@accent = flamingo), (@accent = lavender) {
+                a:visited:not(.flash a) {
+                    color: @pink;
+                }
+            }   
+            & when (@accent = pink) {
+                a:visited:not(.flash a) {
+                    color: @mauve;
+                }
+            }
+            & when (@accent = mauve), (@accent = teal),(@accent = sky) {
+                a:visited:not(.flash a) {
+                    color: @blue;
+                }
+            }
+            & when (@accent = red), (@accent = maroon) {
+                a:visited:not(.flash a) {
+                    color: @peach;
+                }
+            }
+            & when (@accent = peach) {
+                a:visited:not(.flash a) {
+                    color: @red;
+                }
+            }
+            & when (@accent = yellow) {
+                a:visited:not(.flash a) {
+                    color: @maroon;
+                }
+            }
+            & when (@accent = green), (@accent = sapphire) {
+                a:visited:not(.flash a) {
+                    color: @teal;
+                }
+            }
+            & when (@accent = blue) {
+                a:visited:not(.flash a) {
+                    color: @sky;
+                }
+            }
+            & when (@accent = teal) {
+                a:visited:not(.flash a) {
+                    color: @blue;
+                }
+            }
+        }
```

Comment:
please use
```
        a:vistied:not(.flash a) {
              color: @accent-color;
        }
```
instead of using ``& when (@accent = <accent>``

Reply:
> The `& when (@accent = <accent>` is so that when the accent color is a certain color, the visited color is different. I did that so the color remains harmonious but also has some contrast

i guess i'll mark this conversation as resolved

Reply:
> > The `& when (@accent = <accent>` is so that when the accent color is a certain color, the visited color is different. I did that so the color remains harmonious but also has some contrast
> 
> i guess i'll mark this conversation as resolved

Omar, for the 20 millionth time, please learn to use thumbs up in conversations. It benefits everyone involved.

> The `& when (@accent = <accent>` is so that when the accent color is a certain color, the visited color is different. I did that so the color remains harmonious but also has some contrast

Instead we typically use `lighten` or `darken` with 5-10% changes. I know it's weird but its better then repeating what you hvae here across however many userstyles we have now.

Reply:
> Omar, for the 20 millionth time, please learn to use thumbs up in conversations. It benefits everyone involved.
fine, i'll delete the comment and leave a thumbsup

Reply:
> > Omar, for the 20 millionth time, please learn to use thumbs up in conversations. It benefits everyone involved.
> 
> fine, i'll delete the comment and leave a thumbsup

That could be a comment...

Reply:
> Instead we typically use `lighten` or `darken` with 5-10% changes. I know it's weird but its better then repeating what you hvae here across however many userstyles we have now.

Implemented this. Thank you!